### ISSUE TITLE
033

### ISSUE THREAD

**Issue by just-nana** [thendo991/CleanCity#33]
No alt for an image.

Expected: All images should have "alts" for screen readers.
Actual: This image does not have an "alt".

Main index.html file

<img width="1535" height="424" alt="Image" src="https://github.com/user-attachments/assets/09e65ffd-8371-409b-a84e-f322683596a6" />
